src/NUnitFramework/nunitlite-runner/nunitlite-runner.csproj

```diff
@@ -35,7 +20,4 @@
     <ProjectReference Include="..\framework\nunit.framework.csproj" />
     <ProjectReference Include="..\nunitlite\nunitlite.csproj" />
   </ItemGroup>
-
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition="'$(UseWindowsFormsAndWPF)' != 'true'" />
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk.WindowsDesktop" Condition="'$(UseWindowsFormsAndWPF)' == 'true'" />
```

Comment:
Similar question here, I think this helps fix something?

Reply:
In SDKs pre .NET 5 one had to select different Sdk projects if one wanted to use Forms or WPF.
This was [unified in NET 5](https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/5.0/sdk-and-target-framework-change) so that this is no longer needed.
But as our library doesn't use either, I'm not sure why it was ever in there.
This was removed in master (nunit4) a year ago by Rob. Commit hash 3727fce5f031acb0179c591f455b65703747a9d4.